RecoPPS/RPixEfficiencyTools/README.md

```diff
@@ -1,3 +1,47 @@
+## Onboarding
+### Onboarding - how to setup workspace 
+1. login to ssh 
+```
+ssh <username>@lxplus.cern.ch
+# example
+ssh mobrzut@lxpllus.cern.ch
+```
+
+2. Go to your afs directory *(note1: this project was tested with afs not eos - probalby some problems with eos)*

```

Comment:
HTCondor submissions are not allowed from eos, this is the main reason why the setup is done on afs. The full submission procedure could not be tested on eos. Remove the second sentence and write something like 'HTCondor submission required', or 'currently not available on eos'. I actually discovered that there's a trick to do that, but still, it's not fully supported yet.